github/repos_releases.go

```diff
@@ -253,7 +253,7 @@ func (s *RepositoriesService) DownloadReleaseAsset(ctx context.Context, owner, r
 	}
 
 	// TODO: remove custom Accept header when APIs fully launch.
-	acceptHeaders := []string{defaultMediaType, mediaTypeGraphQLNodeIDPreview}
+	acceptHeaders := []string{defaultMediaType}
 	req.Header.Set("Accept", strings.Join(acceptHeaders, ", "))
```

Comment:
This is a good start, but let's simplify the code further. We can make it as simple as it was [before](https://github.com/google/go-github/commit/25b0481acbe9f2123f2b39c3235b156cff6296cd#diff-75909ea88d89029fe24198471fc539aeR228) the `mediaTypeGraphQLNodeIDPreview` header was added to it:

```Go
req.Header.Set("Accept", defaultMediaType)
```

This function operates on binary data, and it's very unlikely its media type will ever need to change. (But if so, it'll be easy to do so then.)

The `// TODO: remove custom Accept header when APIs fully launch.` should be removed too, since it's not applicable (there isn't any preview API we're waiting to fully launch).